src/index.html

```diff
@@ -5,8 +5,220 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:400,500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="container-header">
+      <img src="./images/logo.png" alt="logo" class="logo">
+      <nav class="navbar">
+        <ul class="navbar__list">
+          <li class="navbar__item">
+            <a class="navbar__link is-active" href="#">APPLE</a>
+          </li>
+          <li class="navbar__item">
+            <a class="navbar__link" href="#">SAMSUNG</a>
+          </li>
+          <li class="navbar__item">
+            <a class="navbar__link" href="#">СМАРТФОНЫ&nbsp;И&nbsp;ТЕЛЕФОНЫ</a>
+          </li>
+          <li class="navbar__item">
+            <a class="navbar__link" href="#" data-qa="nav-hover" >НОУТБУКИ&nbsp;И&nbsp;КОМПЬЮТЕРЫ</a>
+          </li>
+          <li class="navbar__item">
+            <a class="navbar__link" href="#">ГАДЖЕТЫ</a>
+          </li>
+          <li class="navbar__item">
+            <a class="navbar__link" href="#">ПЛАНШЕТЫ</a>
+          </li>
+          <li class="navbar__item">
+            <a class="navbar__link" href="#">ФОТО</a>
+          </li>
+          <li class="navbar__item">
+            <a class="navbar__link" href="#">ВИДЕО</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main-content">
+      <section class="main-content__products">
+        <article class="cards" data-qa="card">
+          <img class="card__logo" src="./images/imac.jpeg"  alt="iMac" width="160" height="134">
+          <div class="card__description">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</div>
+          <div class="card__code">Код товара: 195434</div>
+          <div class="rating">
+            <div class="rating__stars">
+              <div class="rating__stars--active"></div>
```

Comment:
якщо є модифікатор має бути і сам клас
```suggestion
              <div class="rating__stars rating__stars--active"></div>
```